generated/json_data/qol.jsonc

```diff
@@ -10504,6 +10561,7 @@ Temporarely unfixing the door because if door lock rando, if there is a blast sh
                         }
                     ],
                     "layers": {
+                        "1": true,
```

Comment:
I'm going to fucking shid my pants miguel

Reply:
I'M SORRRRYYY I FORGOT TO SYNC MY STUFF THE FIRST TIME 😭😭